browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
+    "version": "3.0.1-beta",
+    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
+    "main": "./lib/index.js",
+    "license": "MIT",
```

Comment:
Not Apache 2 like all our other code?

Reply:
This was just the default. I changed it to `Apache-2.0`.